From 826b11e8adea942a7fb63fa54374d5571aab0930 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 5 Feb 2013 16:45:42 +0100
Subject: [PATCH 4/5] dracut.sh: reverting return value change

reverting _get_fs_type() return values of patch
c8d685c9d3860e49f39a9c85ffebbb4c4fec341f
---
 dracut.sh | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/dracut.sh b/dracut.sh
index cfd10a1..23ffaa4 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -766,18 +766,18 @@ if [[ $hostonly ]]; then
 fi
 
 _get_fs_type() (
-    [[ $1 ]] || return 1
+    [[ $1 ]] || return
     if [[ -b $1 ]] && get_fs_env $1; then
         echo "$(readlink -f $1)|$ID_FS_TYPE"
-        return 0
+        return 1
     fi
     if [[ -b /dev/block/$1 ]] && get_fs_env /dev/block/$1; then
         echo "$(readlink -f /dev/block/$1)|$ID_FS_TYPE"
-        return 0
+        return 1
     fi
     if fstype=$(find_dev_fstype $1); then
         echo "$1|$fstype"
-        return 0
+        return 1
     fi
     return 1
 )
-- 
1.8.1.4

